flink-python/src/main/java/org/apache/flink/datastream/runtime/typeutils/python/PythonTypeUtils.java

```diff
@@ -308,6 +366,14 @@ public static TypeSerializer typeInfoSerializerConverter(TypeInformation typeInf
 						.map(f -> typeInfoSerializerConverter(f)).toArray(TypeSerializer[]::new);
 					return new TupleSerializer(Tuple.getTupleClass(tupleTypeInfo.getArity()), fieldTypeSerialzers);
 				}
+
+				if (typeInformation instanceof BasicArrayTypeInfo){
```

Comment:
Do we need to add the corresponding test content in `testTypeInfotoSerializerConverter`